src/services/completions.ts

```diff
@@ -886,9 +888,6 @@ namespace ts.Completions {
             // Completion should work in the brackets
             const tag = getJsDocTagAtPosition(currentToken, position);
             if (tag) {
-                if (tag.tagName.pos <= position && position <= tag.tagName.end) {

```

Comment:
I don't understand why this isn't needed any more.

Reply:
This code causes the completion list of the __jsdoc tag__ to appear in unintended places
It is also the reason why __jsdoc tag__ is listed by `/**|c| */` and `* @type {"@|c|"}`.

 + before (3.9.0-dev.20200414)
![before](https://user-images.githubusercontent.com/26692481/80597123-60949600-8a62-11ea-83b0-2a41357a351d.png)

 + after
![after](https://user-images.githubusercontent.com/26692481/80597131-638f8680-8a62-11ea-9dbc-3445433d479d.png)

Reply:
However, the main purpose of this PR is to __fix regex__, so I will revoke comment out.